examples/05_nossr/entries.gen.tsx

```diff
@@ -0,0 +1,36 @@
+import { createPages } from 'waku';
+import type { PathsForPages } from 'waku/router';
+
+import Layout, { getConfig as Layout_getConfig } from './src/pages/_layout';
+import About, { getConfig as About_getConfig } from './src/pages/about';
+import Index, { getConfig as Index_getConfig } from './src/pages/index';
+import NewHi from './src/pages/new/hi';
+
+const DEFAULT_CONFIG = { render: 'dynamic' } as const;
```

Comment:
@dai-shi is this correct? Do we assume dynamic or static when no getConfig is defined?

Reply:
For pages, yes. For layouts, no.

Reply:
fixed this by removing `DEFAULT_CONFIG` and adding inline on the type generation, but matched the behavior of pages as dynamic and layouts as static